Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json

```diff
@@ -0,0 +1,563 @@
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Jordan C. Ciciliano",
+                "orcid": "https://orcid.org/0000-0003-1427-2608",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Huili Zhu",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Olivia C. MacKenzie",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Julie Trautwein",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Kshitij S. Arora",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mohammad Shahid",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112